draft-rswg-rfc7990-updates.mkd

```diff
@@ -216,13 +214,12 @@ This document has no IANA considerations.
 
 # Security Considerations
 
-Changing the format for RFCs involves modifying a great number of components to publication.
-Understanding those changes and the implications for the entire tool chain is critical so as to avoid unintended consequences that would allow unintended changes to text.
-Unintended changes to text could in turn corrupt a standard, practice, or critical piece of information about a protocol.
+Allowing changes to the definitive versions and publication versions of RFCs introduces two key risks:
 
-The RSWG is responsible for managing the risk of semantic changes that would affect the interpretation of existing and future RFCs.
-Changes to content that has security implications would have security-relevant consequences.
+1. Unintended changes could occur in either the definitive version or publication versions of an RFC as a result of an editing error, or may be introduced into a publication version when it is regenerated from the definitive version.  This may result in the corruption of a standard, practice, or critical piece of information about a protocol, and harm to the reputation of the RFC series.
+2. The reputation of the RFC series may also be adversely affected as a result of new versions being released for trivial reasons, leading to consumers of RFCs feeling they have wasted effort in reviewing or processing new versions, or by new versions being released too often, leading to a loss of trust in the stability of the series.
```

Comment:
As noted on the list, I object to establishing this as a goal.

Especially in a security considerations section.